commerce-service/service.xml

```diff
@@ -249,7 +249,7 @@
 		<!-- Other fields -->
 
 		<column name="commerceAccountId" type="long" />
-		<column name="commerceCurrencyId" type="long" />
+		<column name="commerceCurrencyCode" type="String" />
```

Comment:
`commerceCurrencyCode` should be sorted lower now..... at least below `shippingAddressId`, though it could potentially go lower if you like a lower spot.